database_functions/functions.py

```diff
@@ -14630,49 +14672,10 @@ def get_user_by_email(cnx, database_type, email):
     finally:
         cursor.close()
 
-def create_oidc_user(cnx, database_type, email, fullname, base_username):
+def create_oidc_user(cnx, database_type, email, fullname, username):
     cursor = cnx.cursor()
     try:
-        print(f"Starting create_oidc_user for email: {email}, fullname: {fullname}, base_username: {base_username}")
```

Comment:
This logic seemed to be present in two places, here and in the OIDC callback, so for simplicity I just removed it here and only handled in the OIDC callback to support the case where the username is preset and we _don't_ want to try alternatives.